src/explore-education-statistics-frontend/server.js

```diff
@@ -81,20 +83,42 @@ async function startServer() {
     return `${input.slice(0, -pattern.length)}${replacement}`;
   }
 
-  // Redirect URLs with trailing slash to equivalent without slash with 301
   server.use((req, res, nextFunc) => {
+    if (
+      req.url === '/' ||
+      req.url.startsWith('/assets') ||
+      req.url.startsWith('/_next')
+    ) {
+      nextFunc();
+      return undefined;
+    }
+
     let newUri = req.url;
+    // Redirect URLs with trailing slash to equivalent without slash with 301
     newUri = replaceLastOccurance(newUri, '/', '');
+
+    // These are generated by spam sites. We could redirect them too to reduce Google's indexed 404 pages,
+    // but it seems current advice is actually to ignore these:
+    // https://support.google.com/webmasters/thread/253569816/google-crawler-adding-a-1000-to-the-end-of-a-ton-of-urls?hl=en
+    // newUri = replaceLastOccurance(newUri, '/1000', '');
+
+    // TODO: Remove these redirects after Google's index is updated (will happen within 6 months)
```

Comment:
Is there a jira ticket we can reference for this to do?

Reply:
Good shout, have created (& referenced) https://dfedigital.atlassian.net/browse/EES-4979